node/src/components/network/tests2.rs

```diff
@@ -0,0 +1,238 @@
+use std::{
```

Comment:
nitpick...I'm not loving this trend I've noticed of suffixing filenames vs giving them more verbose names.

Reply:
Good point. I'll rename it. I actually would want to potentially `test2` with `test`, as they are similar enough, the separate implementation was done in the name of expediency.